src/app/views/river-detail/river-creator.vue

```diff
@@ -210,16 +210,17 @@ export default {
           });
           this.$router.push(`/river-detail/${result.data.reachUpdate.id}`);
         } else {
+
           this.$store.dispatch("Global/sendToast", {
             kind: "error",
             title: "Error",
-            subtitle: "Failed to Create New Reach.",
+            subtitle: "Failed to Create New Reach: "+(Object.getOwnPropertyNames(result.errors).map(x=>result.errors[x]?.message ?? '').join(','))
```

Comment:
I think you want `Object.keys` here not `Object.getOwnPropertyNames`, right? they probably behave the same in most situations here given we know more or less what's coming from the API, but `getOwnPropertyNames` could possibly include things you're not intending to include?

Reply:
got it. also added support for axios's message on a hard error.